Add 19th week (#279)
packit/packit.dev: content/posts/May-2021.md

  lead to better responsiveness
  ([packit-service#1059](https://github.com/packit/packit-service/pull/1059),
  [deployment#202](https://github.com/packit/deployment/pull/202)).

## Week 19 (May 10th - May 14th)

- Jirka added support for creating Bugzilla bugs for newly created MRs on GitLab
  ([packit-service#1087](https://github.com/packit/packit-service/pull/1087)).

> - The installation instructions for COPR build that has not finished or was
>   unsuccessful will not be displayed on the results page

Reviewer: You're mixing present perfect (`has not finished`) and past simple (`was unsuccessful`).
I'd simplify it with:
```suggestion
- The installation instructions for failed or unfinished COPR builds are not be displayed on the results page
```
